src/views/virtualmachines/details/components/disk/utils/VirtualMachineDisksTabUtils.ts

```diff
@@ -60,11 +60,10 @@ export const getDiskInterface = (disk: V1Disk): string => disk[getDiskDrive(disk
 
 export const getPrintableDiskInterface = (disk: V1Disk): string => {
   const diskInterface = getDiskInterface(disk);
-  return diskInterface === 'virtio'
-    ? diskInterface
-    : diskInterface === 'scsi' || diskInterface === 'sata'
-    ? diskInterface.toUpperCase()
-    : '';
+  if (diskInterface === 'scsi' || diskInterface === 'sata') {
+    return diskInterface.toUpperCase();
+  }
+  return diskInterface ?? '';
```

Comment:
this change will conflict have with [1], I can insert this change to [1] instead to have this here, and it's also not really related to this PR :)